docs/_config.yml

```diff
@@ -76,6 +76,9 @@ sphinx:
       nbformat:
         - "https://nbformat.readthedocs.io/en/latest"
         - null
+      sphinx-panels:
+        - https://sphinx-panels.readthedocs.io/en/latest/
```

Comment:
```suggestion
        - https://sphinx-panels.readthedocs.io/en//sphinx-book-theme/
```

Reply:
? this seems not quite right

Reply:
fixed

Reply:
is there a reason that we wanna pin to a specific branch of sphinx-panels? Is that something we'll revert back to `master` at some point?

Reply:
nope its my awesome system for showing of the docs in different themes (particularly because sphinx-panels is very CSS dependent):

<img width="328" alt="image" src="https://user-images.githubusercontent.com/2997570/93390536-c04eda80-f865-11ea-9a38-7f673c11b7dc.png">

Reply:
master is in `rtd_sphinx_theme`